### ISSUE TITLE
Ensure oatpp.io is consistently available

### ISSUE THREAD

**Issue by RenFraser** [oatpp/website#22]
Most times that I access the oatpp website, I'm met with an empty response (screenshot attached). This makes it difficult when trying to access the docs as users have to keep refreshing their web page until it works. When developing using the Oatpp library, having consistently available docs is important. My gut feel is that it's a hosting issue not being able to keep up with demand. 

![oatpp_down](https://user-images.githubusercontent.com/19373652/166341866-cc548b32-215b-46ec-bb69-db8fd1536a50.PNG)

**Comment by bamkrs** [comment]
Where are you accessing oatpp.io from?

**Comment by RenFraser** [comment]
That's right, my colleagues and I are in Australia. How is it hosted at present?
